modules/ROOT/pages/troubleshoot-saml-assertions-task.adoc

```diff
@@ -34,10 +34,10 @@ https://www.mulesoft.com/support-login
 
 == Example: Chrome Developer Tools SAML Response Element
 
-The following image shows a SAML response element in Chrome developer tools as a result of logging in to Anypoint Platform using SSO.
+The following image shows a SAML response element in Chrome developer tools as a result of signing in to Anypoint Platform using SSO.
 
-.SAML response element after logging in to Anypoint Platform in Google Chrome.
+.SAML response element after signing in to Anypoint Platform in Google Chrome.
 image::iam-chromedevtools.png[]
 
-In the *Network* tab (1), the `receive-id` line (2) indicates that you attempted to log in to Anypoint Platform using single sign-on through your identity provider.
-Because the login is successful, you can expand `receive-id` and access form data that contains your SAML response element (3). Note that unsuccessful login attempts still generate SAML response elements.
+In the *Network* tab (1), the `receive-id` line (2) indicates that you attempted to sign in to Anypoint Platform using single sign-on through your identity provider.
```

Comment:
single signout had no hyphen. Should these be consistent?

Reply:
Should this be signin rather than sign-on?

Reply:
So, this is SSO and I left it as single sign-on, since that seems to be the industry standard?